almalinux-9-vagrant.pkr.hcl

```diff
@@ -142,7 +183,7 @@ build {
     ansible_env_vars = [
       "ANSIBLE_PIPELINING=True",
       "ANSIBLE_REMOTE_TEMP=/tmp",
-      "ANSIBLE_SSH_ARGS='-o ControlMaster=no -o ControlPersist=180s -o ServerAliveInterval=120s -o TCPKeepAlive=yes'"
```

Comment:
Please roll back this change, since it affects all of the cloud-images build processes. 

If you would like to propose the Ansible SSH options, create a separate PR/issue and justify it.